src/core/include/openvino/op/mvn.hpp

```diff
@@ -116,6 +116,10 @@ class OPENVINO_API MVN : public Op {
 
     std::shared_ptr<Node> clone_with_new_inputs(const OutputVector& new_args) const override;
 
+    bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
```

Comment:
this overload is deprecated, let's override:
```c++
    virtual bool evaluate(ov::TensorVector& output_values, const ov::TensorVector& input_values) const;
```

Reply:
Done.